asset-tracking-location-replaying/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "asset-tracking-location-replaying",
+  "version": "1.0.0",
+  "description": "CLI tool that replays location updates",
+  "main": "index.ts",
+  "scripts": {
+    "start": "./node_modules/ts-node/dist/bin.js src/index.ts"
```

Comment:
I'm not 100% sure about the setup here. It feels wrong for us to be running the typescript compiler every time someone runs the CLI. I think ideally we should have a build script and distribute the built javascript. In any case, if we leave it like this then the compiler would fail to run with the production dependencies because it needs @types/node to compile. I don't mind leaving it like this for now since it's early stages but we would need to move @types/node out of devDependencies for this to work.

Reply:
Also ts-node exports a bin property so it's better to use `ts-node` instead of `./node_modules/ts-node/dist/bin.js`.

Reply:
I've got rid of the `ts-node` and just use `tsc` to translate TypeScript into JavaScript :wink: https://github.com/ably/ably-asset-tracking-common/pull/25/commits/aa70c1c9fdacea467eccae1d8d4049fbb0336d3a